recipes/genesis2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "genesis2" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/Genesis-1.3-Version2/archive/v{{ version }}.tar.gz
+  sha256: 0c8be1fd80cfa8339e3e40b5e798ec596d339e9b743c46a187321a71b9766e12
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('fortran') }}
+
+test:
+  commands:
+    - genesis2
+
+about:
+  home: hhttps://github.com/slaclab/Genesis-1.3-Version
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
Would suggest adding this to the GitHub repo as well. Then we can just grab it directly from the source in the future

Reply:
As soon as we get this recipe going I will add the license and cut a new release of the package to include it.